lib/Cro/Tools/CLI.pm6

```diff
@@ -195,15 +195,15 @@ multi MAIN('link', 'rm', $from-service-id, $to-service-id, $to-endpoint-id?) {
     rm-link($from-service-id, $to-service-id, $to-endpoint-id);
 }
 
-multi MAIN('run') {
-    run-services();
+multi MAIN('run', Str:D :$host? = 'localhost') {
+    run-services(:$host);
 }
 
-multi MAIN('run', *@service-name) {
-    run-services(filter => any(@service-name));
+multi MAIN('run', *@service-name, Str:D :$host? = 'localhost') {
+    run-services(filter => any(@service-name), :$host);
 }
 
-multi MAIN('trace', *@service-name-or-filter) {
+multi MAIN('trace', *@service-name-or-filter, Str:D :$host? = 'localhost') {
```

Comment:
Named parameters are optional by default (and providing a default also implies optional), so `Str:D :$host = 'localhost'` would suffice here.

Reply:
Okay, I've removed the "?", I'm not sure why I was thinking that was necessary (I know better!).